packages/dependency-extraction-webpack-plugin/test/fixtures/polyfill-magic-comment-minified/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '../../..' );
+
+module.exports = {
+	optimization: {
+		minimize: true,
```

Comment:
I was contemplating why the `version` didn't change for existing test cases, where the `wp-polyfill` was listed. However, I figured out we didn't use modification before.